internal/batches/service/service.go

```diff
@@ -43,10 +43,12 @@ func New(opts *Opts) *Service {
 // The reason we ask for batchChanges here is to surface errors about trying to use batch
 // changes in an unsupported environment sooner, since the version check is typically the
 // first thing we do.
-const sourcegraphVersionQuery = `query SourcegraphVersion {
+
+const getInstanceInfo = `query InstanceInfo {
 	site {
 		productVersion
 	}
+	maxUnlicensedChangesets
 	batchChanges(first: 1) {
```

Comment:
Nit: It seems we don't use this fields so we should be able to remove it.

Reply:
I know its not related to your work here but its not used so I thought to call it out.

Reply:
It's mentioned in the docstring up above, I think we use this as a hack to get better error feedback if you try to run a batch change on dotcom or another unsupported environment.

Reply:
Oh! I missed that. Thanks